src/main/java/org/intellij/sdk/codesync/Database.java

```diff
@@ -0,0 +1,110 @@
+package org.intellij.sdk.codesync;
+
+import org.intellij.sdk.codesync.DataClass.TransformFileToDB;
+
+import java.io.File;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.intellij.sdk.codesync.Constants.DATABASE_PATH;
+import static org.intellij.sdk.codesync.Constants.USER_FILE_PATH;
+
+public class Database {
+
+    private static Connection connection = null;
+
+    public static void initiate() {
+
+        try{
+            File file = new File(DATABASE_PATH);
+            if(!file.exists()){
+                Class.forName("org.sqlite.JDBC");
+                String connectionString = "jdbc:sqlite:" + DATABASE_PATH;
+                connection = DriverManager.getConnection(connectionString);
+                createTable();
+                TransformFileToDB transformFileToDB = new TransformFileToDB();
+                transformFileToDB.readUsersInFile();
+            }else {
+                Class.forName("org.sqlite.JDBC");
+                String connectionString = "jdbc:sqlite:" + DATABASE_PATH;
+                connection = DriverManager.getConnection(connectionString);
+                createTable();
+            }
+
+        }catch (Exception exception) {
+            System.out.println("Database connection error: " + exception.getMessage());
+        }
+    }
+
+    public static void createTable(){
+        try {
+            Statement statement = connection.createStatement();
+            statement.executeUpdate(
+                    "CREATE TABLE IF NOT EXISTS user " +
```

Comment:
We can make this generic
1- We can put CREATE query in a `.sql` file and read the query from there. 
2- It will become `createTable(tablesSchema);`
3- We can keep updating tablesSchema to add new tables and stuff and we won't have to change this code later
Thoughts @syedgulahmed ?

Reply:
I guess I just need to remove this method because executeUpdate is doing similar job after accepting query string.